mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -639,4 +646,10 @@ void registrationsShouldNotHandleEventsAfterStop() throws Exception {
         }
 
     }
+
+    private void assertThatListenerReceiveOneEvent(MailboxListener listener) {
+        assertThatThrownBy(() -> verify(listener, after(THIRTY_SECONDS.toMillis()).never()).event(EVENT))
```

Comment:
why not using calmly await?

Reply:
The listener doesn't expose methods which would allow us to check if the event method was called.

Reply:
I dont think you need that, this I what I think in my mind:
```java
calmlyAwait
    .until(() -> verify(listener).event(EVENT));
```